treebuilder.ts

```diff
@@ -103,7 +103,7 @@ function findDependencies(fileName) {
 
 export function getGitStagedFiles(scope = '') {
   return new Promise((resolve, reject) => {
-    cmd.get('git status --no-renames -s', (err, data, stderr) => {
+    cmd.get('git status --porcelain', (err, data, stderr) => {
```

Comment:
This flag seems to be the way to go when parsing git status. Not sure it's super important in this very case, but I thought we might as well use it for what it's designed.